web/server/entrypoint.go

```diff
@@ -216,6 +209,37 @@ func (s *robotServer) createWebOptions(cfg *config.Config) (weboptions.Options,
 	return options, nil
 }
 
+func (s *robotServer) updateLoggerRegistry(cfg *config.Config) {
+	var combinedLogCfg []logging.LoggerPatternConfig
+	if cfg.LogConfig != nil {
+		combinedLogCfg = append(combinedLogCfg, cfg.LogConfig...)
+	}
+
+	for _, serv := range cfg.Services {
```

Comment:
We still rely on ordering here. `LogConfig` processed before `LogConfiguration`s means `LogConfiguration`s take precedence over `LogConfig`.

Reply:
@cheukt you mean a helper function that just returns a `[]logging.LoggerPatternConfig`? I like that idea. `logging` feels like the right place to have such a helper function.

Reply:
either one that returns `[]logging.LoggerPatternConfig` or one that acts upon the registry and updates it as well

Reply:
Good idea; moved it to be a helper in `config`. I agree that `logging` would be a better package, but `config` already depends on `logging`, so there would be an import cycle.

Reply:
> so there would be an import cycle

the function could take in just the `resource.Config` and `logging.Config` fields to avoid the import cycle

Reply:
> take in just the `resource.Config`

Unfortunately `logging` depending on `resource` would also be an import cycle. Most of our packages depend on `logging`, so helpers _in_ `logging` that reference other packages will usually result in cycles.